src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { StateContext } from '../../reducer/store';
+
+export const Notifications: React.FC = () => {
+  const {
+    notificationMessage,
+    notificationType,
+    notificationData,
+  } = useContext(StateContext);
```

Comment:
The useContext hook is used incorrectly. You should use it with a specific context that you want your component to subscribe to. Also, it seems the StateContext does not provide the required data directly.